src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -232,6 +250,122 @@ def end_string(self):
         return "omp end single"
 
 
+class OMPMasterDirective(OMPSerialDirective):
+    '''
+    Class representing an OpenMP MASTER directive in the PSyclone AST.
+
+    :param list children: list of Nodes that are children of this Node.
+    :param parent: the Node in the AST that has this directive as a child.
+    :type parent: :py:class:`psyclone.psyir.nodes.Node`
+
+    '''
+    def __init__(self, children=None, parent=None):
```

Comment:
Done - I left the documentation though maybe I shouldn't have? Not sure.

Reply:
We delete the parameter docstrings because its hard to remember to update them when changing the parent class. I will delete them myself before pushing to master.